packages/block-editor/src/store/selectors.js

```diff
@@ -2005,6 +2005,11 @@ export const getInserterItems = createSelector(
 				isDisabled: false,
 				utility: 1, // Deprecated.
 				frecency,
+				syncStatus: reusableBlock.meta?.sync_status,
+				content: reusableBlock.content.raw,
+				blocks: parse( reusableBlock.content.raw, {
+					__unstableSkipMigrationLogs: true,
+				} ),
```

Comment:
What does this code do?

The `parse` might need to be assessed in terms of performance impact. I think patterns are generally pre-parsed everywhere.

A concern is that `syncStatus` is a very block specific thing, while this is quite a generic selector that maybe shouldn't know about block specifics.

Reply:
yeh, have [moved this just to the patterns tab](https://github.com/WordPress/gutenberg/pull/51144/commits/9ec8a98f54df2787842742ed1a69771228ba80b5) where it is actually needed

Reply:
I wonder if the filtering out of synced patterns might also be better to do in the selector, that way you won't have to return `sync_status` in the data.

Reply:
I have moved the [sync status filter to the selector](https://github.com/WordPress/gutenberg/pull/51144/commits/ee75c262b10ec33b7e4b24a57a2cc3fa26a57a81)